x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -13,11 +13,26 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const log = getService('log');
   const security = getService('security');
   const testSubjects = getService('testSubjects');
+  const es = getService('es');
   const find = getService('find');
   const browser = getService('browser');
 
+  async function clearAllApiKeys() {
+    const existingKeys = await es.security.queryApiKeys();
+    if (existingKeys.count > 0) {
+      Promise.all(
+        await existingKeys.api_keys.map(async (key) => {
```

Comment:
'await' has no effect here, but it must be before `Promise.all(..)`

Reply:
OK, understood. Fixed.